tests/smoketest.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+set -eux
```

Comment:
I would also add `-o pipefail` here as this will cause it to stop if any command within a pipeline fails:
```suggestion
set -eux -o pipefail
```

Reply:
👍